Directory.Packages.props

```diff
@@ -29,7 +29,6 @@
     <SystemSecurityCryptographyPkcsVersion Condition="'$(SystemSecurityCryptographyPkcsVersion)' == ''">9.0.6</SystemSecurityCryptographyPkcsVersion>
     <SystemSecurityCryptographyProtectedDataVersion Condition="'$(SystemSecurityCryptographyProtectedDataVersion)' == ''">9.0.6</SystemSecurityCryptographyProtectedDataVersion>
     <MicrosoftVisualStudioSolutionPersistenceVersion Condition="'$(MicrosoftVisualStudioSolutionPersistenceVersion)' == ''">1.0.52</MicrosoftVisualStudioSolutionPersistenceVersion>
-    <MicrosoftVisualStudioSdkForApex>18.0.1799-preview.1</MicrosoftVisualStudioSdkForApex>
```

Comment:
This commit didn't remove where this property was being used: https://github.com/search?q=repo%3ANuGet%2FNuGet.Client%20MicrosoftVisualStudioSdkForApex&type=code

Reply:
The search results also show it was used in NuGet.OptProf.csproj, but isn't changed in the latest commit as of the time I'm writing.

Reply:
Thanks @zivkan. I'm fixing that again. I had a bunch of issues with the rebase merge so I had to fix that and I think I lost the update I made in NuGet.OptProf.csproj. I'll push a change with that fixed.